posthog/caching/web_analytics_warming.py

```diff
@@ -0,0 +1,216 @@
+    "Number of stale web queries present",
+    ["team_id"],
+)
+PRIORITY_WEB_QUERIES_COUNTER = Counter(
+    "posthog_cache_warming_priority_web_queries",
+    "Number of priority web queries warmed",
+    ["team_id", "normalized_query_hash", "is_cached"],
+)
+
+
+def teams_enabled_for_web_analytics_cache_warming() -> list[int]:
+    enabled_team_ids = []
+
+    for team_id, organization_id, uuid in Team.objects.values_list(
+        "id",
+        "organization_id",
+        "uuid",
+    ).iterator(chunk_size=1000):
+        enabled = posthoganalytics.feature_enabled(
+            "web-analytics-cache-warming",
+            str(uuid),
+            groups={
+                "organization": str(organization_id),
+                "project": str(team_id),
+            },
+            group_properties={
+                "organization": {
+                    "id": str(organization_id),
+                },
+                "project": {
+                    "id": str(team_id),
+                },
+            },
+            only_evaluate_locally=True,
+            send_feature_flag_events=False,
+        )
+
+        if enabled:
+            enabled_team_ids.append(team_id)
+
+    return enabled_team_ids
+
+
+def queries_to_keep_fresh(team_id: int, days: int = 7, minimum_query_count: int = 10) -> list[dict]:
+    results = sync_execute(
+        """
+        SELECT
+            team_id,
+            query_json_raw,
+            COUNT(*) AS query_count,
+            normalizedQueryHash(query) as normalized_query_hash
+        FROM (
+            SELECT
+                team_id,
+                JSONExtractRaw(log_comment, 'query') AS query_json_raw,
+                query
+            FROM metrics_query_log_mv
+            WHERE
+                timestamp >= now() - INTERVAL %(days)s DAY
+                AND team_id = %(team_id)s
+                AND query_type IN (
+                    'stats_table_query',
+                    'web_goals_query',
+                    'web_overview_preaggregated_query',
+                    'web_overview_query',
+                    'web_vitals_path_breakdown_query'
+                )
+        ) AS sub
+        GROUP BY
+            team_id,
+            query_json_raw,
+            normalized_query_hash
+        HAVING query_count >= %(minimum_query_count)s
+        ORDER BY
+            query_count DESC
+        """,
+        {"team_id": team_id, "days": days, "minimum_query_count": minimum_query_count},
+    )
+
+    logger.info("Frequent web analytics queries", team_id=team_id, query_count=len(results))
+
+    return [
+        {
+            "team_id": result[0],
+            "query_json": json.loads(result[1]),
+            "query_count": result[2],
+            "normalized_query_hash": result[3],
+        }
+        for result in results
+    ]
+
+
+@shared_task(ignore_result=True, expires=60 * 15)
+def schedule_web_analytics_warming_for_teams_task():
+    team_ids = largest_teams(limit=10)
```

Comment:
How safe are we that we can do this without killing the cluster? 

If the queries have pre-aggregation enabled, this will likely be fast... if they don't, I am concerned we might be sending a bunch of queries that won't be used. 

We will eventually need to find our own logic here because web analytics queries will likely have less granularity than Product Analytics insights. 

So, to make this whole comment more actionable: wdyt if we start only with the FF strategy for now?